laura/portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/chart")
+public class ChartServlet extends HttpServlet {
+    private int[] frequency = {0,0,0,0,0};
+
+    public void doPost (HttpServletRequest request, HttpServletResponse response) throws IOException {
+        int pickedNmb = Integer.parseInt(request.getParameter("picked-cereal"));
+        ++ frequency[pickedNmb];
```

Comment:
You don't need to get the value, so it's better to use more common variant ``` frequency[pickedNmb]++; ```  
Increment and get (``` ++frequency[pickedNmb]; ```) is used only when you need to read the new value
immediately.

That's not wrong, just a code convention.